src/napari_omero/plugins/loaders.py

```diff
@@ -99,6 +99,14 @@ def load_image_wrapper(image: ImageWrapper) -> list[LayerData]:
     # contrast limits range ... not accessible from plugin interface
     # win_min = channel.getWindowMin()
     # win_max = channel.getWindowMax()
+
+    # get json metadata
+    from omero_marshal import get_encoder
```

Comment:
Not sure why ruff didn't auto-fix this, but the import should be at the top with the others.

Reply:
fixed in c9cc4d1a4714f9b60e76397ee0d9199f13d95a2c